magenta/interfaces/midi/magenta_midi.py

```diff
@@ -45,22 +45,27 @@
     'phrase_bars',
     None,
     'The number of bars of duration to use for the call and response phrases. '
-    'If none, `end_call_control_number` must be specified.')
```

Comment:
I think actually `control_number` is much clearer than `control` because it could also refer to value. For me hacking things, I found `number` to make things clearer.